package.json

```diff
@@ -196,8 +196,9 @@
     "electron-devtools-installer": "^3.2.0",
     "electron-updater": "^4.6.1",
     "electron-window-state": "^5.0.3",
-    "filehound": "^1.17.4",
+    "filehound": "^1.17.5",
     "fs-extra": "^9.0.1",
+    "glob": "^7.2.0",
```

Comment:
I don't think you use this dependency in this PR.

Reply:
Removed